### ISSUE TITLE
Can't set timebase to larger than ns

### ISSUE THREAD

**Issue by pdbj** [sstsimulator/sst-core#825]
New Issue for sst-core
----------------------
1 - Detailed description of problem or enhancement
Setting the timebase to larger than ns causes a crash during initialization

2 - Describe how to reproduce the issue
```
$ sst --timebase=1us ...
...
FATAL:  SST Core: Error:  Attempting to get TimeConverter for a time (1 ns) which has too small of a period to be represented by the timebase (1 us)
SST Fatal Backtrace Information:
    0 : sst(_ZNK3SST6Output5fatalEjPKcS2_iS2_z+0x499) [0x5111b7]
    1 : sst(_ZN3SST8TimeLord16getTimeConverterERKNS_11UnitAlgebraE+0x8dc) [0x527918]
    2 : sst(_ZN3SST8TimeLord16getTimeConverterERKSs+0x13a) [0x527bee]
    3 : sst(_ZN3SST8TimeLord4initERKSs+0x85) [0x527d69]
    4 : sst(main+0x853) [0x4bcb2d]
    5 : /lib64/libc.so.6(__libc_start_main+0xf5) [0x2aaaad3c1555]
    6 : sst() [0x4ba7f7]
```

3 - Diagnosis

simulation.cc:211: `Simulation_impl::Simulation_impl()`
```
    if ( my_rank.thread == 0 ) {
        m_exit = new Exit(num_ranks.thread, timeLord.getTimeConverter("100ns"), num_ranks.rank == 1);
    }
```

4 - Suggested fix
Change the `new Exit` call to

```
        m_exit = new Exit(num_ranks.thread, timeLord.getTimeConverter(cfg->timebase()), num_ranks.rank == 1);
```

**Comment by feldergast** [comment]
Thanks.  I'll make the change.  However, the error message gives a time of 1ns, not 100ns (as in the new Exit call), so there is probably an issue somewhere else as well.  For example, the TimeLord creates TimeConverters for 1ns, 1us and 1ms (which may be where the above error is coming from??); we'll need to detect that those are representable with the set time base before creating them.  I'll add a core test with a higher timebase and track down all the places things break.

**Comment by feldergast** [comment]
Fixed in #828
